indra/newview/skins/default/xui/en/strings.xml

```diff
@@ -2699,6 +2699,9 @@ If you continue to receive this message, please contact Second Life support for
 	<string name="IMMainland">mainland</string>
 	<string name="IMTeen">teen</string>
 
+    <!-- floater emoji picker -->
+    <string name="NoEmojiForFilter">No emoji found for '[FILTER]'</string>
```

Comment:
This sting is specific for a signle floater, shouldn't it be declared in emoji loater's xml?

Reply:
> This sting is specific for a signle floater, it's preferable to declared in floater_emoji_picker.xml.

It seems it is placed in the right place
![image](https://github.com/secondlife/viewer/assets/124201357/8f333ef6-2ffe-472d-9dcd-4f927113e684)

Reply:
> but I didn't find the way to define a string with parameters in the floater

example:
```
						args["[MINUTES]"] = llformat("%d", (seconds/60));
						std::string buf = parent_floater->getString ("Minutes", args);
```

Reply:
> This Emoji picker already has couple strings declared in own xml file

Agree, it's better to keep them in one place.

Reply:
> example:

Moved, thanks